its/core-it-support/core-it-plugins/maven-it-plugin-active-collection/src/main/java/org/apache/maven/plugin/coreit/CheckThreadSafetyMojo.java

```diff
@@ -95,7 +95,10 @@ public void run() {
                     getLog().info("[MAVEN-CORE-IT-LOG] Thread " + this + " uses " + tccl);
                     Thread.currentThread().setContextClassLoader(tccl);
                     while (go.isEmpty()) {
-                        // wait for start
+                        try {
+                            Thread.sleep(100); // wait for the start
```

Comment:
```suggestion
                            MILLISECONDS.sleep(100); // wait for the start
```
https://docs.oracle.com/javase/9/docs/api/java/util/concurrent/TimeUnit.html#sleep-long-